### DISCUSSION TITLE
Opening a webpage from a link in Trilium

### DISCUSSION THREAD

**Discussion by ecodiv** [TriliumNext/Trilium#2318] [Q&A]
Clicking on a webpage link in Trilium used to open the webpage in my default browser. In the latest version, it opens in a new instance of Trilium (or I guess I should say it opens in Electron) like a kind of popup. No way to close it directly. Is this a deliberate choice, and if so, is there an option to set it back to opening webpages with the default browser?

**Comment by zadam** [comment]
Hi, this is not intentional. Could you describe more closely how do you open the link and maybe a screenshot (or ideally a video) of this?

**Reply by ecodiv** [reply]
Hi, not sure what happened, but today after start up, the webpages open in my browser again. So it was a glitch I guess, sorry for the noise. But if I happen to see this behavior again, I'll make a screenshot.
